### PR TITLE
Commonly used token operations like minting, getting weth, transferring tokens

### PR DESCRIPTION
**Pull request by agatsoh**
…ng tokens on private or testnet environment

### CODE REVIEW

**Comment by agatsoh** [comment]
Sample commands
```
python token_ops.py mint --rpc-url http://127.0.0.1:8545 --private-key ~/priv_chain/blkchain1/keystore/private_net_address --token-address 0x2feEd0E54238051dddCc01bF3960B143e887a9B7 --amount 1000

python token_ops.py weth --rpc-url http://127.0.0.1:8545 --private-key ~/priv_chain/blkchain1/keystore/private_net_address --token-address 0xdf048aa8cbA44f9590F888BAb5e5AC78AAb503C8 --amount 1000

python token_ops.py transfer --rpc-url http://127.0.0.1:8545 --private-key ~/priv_chain/blkchain1/keystore/private_net_address --token-address 0xdf048aa8cbA44f9590F888BAb5e5AC78AAb503C8 --amount 1000 --destination 0x7ba5f1c08548f80d52856c21e87fcca05c5e40e3

python token_ops.py balance --rpc-url http://127.0.0.1:8545 --token-address 0xdf048aa8cbA44f9590F888BAb5e5AC78AAb503C8 --address 0xb8eb60F2E45667c9B2cFf861b82656452659C6dE

python token_ops.py balance --rpc-url https://rpc.slock.it/goerli --token-address 0x610f3c3C1998FAd6A659A9f5Bb83962DA27eAf1d --address 0x80Cf7AcFde7Cb110675cD7B478c167cF1A5b2f0e
```

**Comment by pirapira** [comment]
Will you, in `README.rst`, describe how to use the script?  Maybe after `Deployment on a testnet`.

**Comment by agatsoh** [comment]
@pirapira I really need your help to write tests for this I am really clueless. I need some guidance regarding this.

**Comment by pirapira** [comment]
@agatsoh I would mock the web3 object, but I think even without tests these scripts are good additions.

Later I'll paste a link showing how to mock the web3 object.

**Comment by pirapira** [comment]
* You can test the TokenOperations class like https://github.com/raiden-network/raiden-contracts/blob/6d64c0260bfbab04479cb65cb0a0d7180daebf71/raiden_contracts/tests/test_deploy_data.py#L132 Basically by passing a mock web3 to it.

**Comment by pirapira** [comment]
* You can test the main function like this https://github.com/raiden-network/raiden-contracts/blob/6d64c0260bfbab04479cb65cb0a0d7180daebf71/raiden_contracts/tests/test_deploy_script.py#L787 Basically by mocking TokenOperations class.

**Comment by pirapira** [comment]
These are the techniques that I know.  Maybe @karlb knows a better way to test these scripts.

**Comment by karlb** [comment]
Using the `web3` fixture to execute blockchain calls via EthereumTester should work for these scripts too, shouldn't it?

**Comment by agatsoh** [comment]
I will try all your recommendations, thanks a lot !!!!

**Comment by pirapira** [comment]
Please run `make format`.

I can figure out the tests later.

File: raiden_contracts/utils/token_ops.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+from web3.middleware import construct_sign_and_send_raw_middleware, geth_poa_middleware
+from raiden_contracts.constants import CONTRACT_CUSTOM_TOKEN
+from raiden_contracts.contract_manager import ContractManager, contracts_precompiled_path
+from raiden_contracts.utils.private_key import get_private_key
+from raiden_contracts.utils.signature import private_key_to_address
+from raiden_contracts.utils.transaction import check_successful_tx
+
+
+class TokenOperations:
+    def __init__(self, rpc_url: str, private_key: str, wait: int = 10):
```
Comment (luehrsFred) [inline raiden_contracts/utils/token_ops.py]: I would suggest adding a possible password file as well. What do you think?

Comment (agatsoh) [reply raiden_contracts/utils/token_ops.py]: Sure let me add it.

**Comment by pirapira** [comment]
This script seems useful.  We can improve it onwards.

**Pull request merged**